pkg/operator/webhooksupportabilitycontroller/degraded_webhook.go

```diff
@@ -101,8 +101,15 @@ func (c *webhookSupportabilityController) assertConnect(ctx context.Context, ref
 		port = fmt.Sprintf("%d", *reference.Port)
 	}
 	rootCAs := x509.NewCertPool()
-	if len(caBundle) > 0 {
+	switch {
+	case net.JoinHostPort(host, port) == "kubernetes.default.svc:443":
+		// ignore any caBundle specified and get one from the service account
```

Comment:
that leaves caBundle field an ignored field for kubernetes.default.svc, right?

Reply:
Correct. This matched the behavior in the apiserver when calling the webhook.